commands/version/index.js

```diff
@@ -556,6 +579,35 @@ class VersionCommand extends Command {
 
     return gitPush(this.gitRemote, this.currentBranch, this.execOpts);
   }
+
+  createGitHubReleases() {
+    this.logger.info("github", "Creating GitHub releases...");
+
+    const client = createGitHubClient();
+    const repo = parseGitUrl(gitRepoUrl());
```

Comment:
I think it would be clearer if `@lerna/github-client` exposed a `parseGitRepo()` method that returned this repo object, rather than embed the `git config --get ...` stuff in the version command. Does that make sense?

(it would also make unit testing easier with more expressive mocking)

Reply:
Yup, totally can do.